docs/enterprise/deployments/deploy-on-kubernetes/installation.md

```diff
@@ -1,8 +1,52 @@
 ---
-keywords: [Installation]
-description: Deploy GreptimeDB on Kubernetes
+keywords: [Kubernetes deployment, GreptimeDB Enterprise, install GreptimeDB, start GreptimeDB, private docker registry, helm chart]
+description: Steps to install GreptimeDB Enterprise on Kubernetes, including obtaining images and starting GreptimeDB.
 ---
 
-# Installation
+# Deploy GreptimeDB Cluster
 
-TODO
\ No newline at end of file
+GreptimeDB Enterprise is released as docker images.
```

Comment:
Are we really want to expose these implementation details to public docs? @sunng87 
I don't see them in other products.

Reply:
It's OK. iDB has similar content